webui/src/components/form/InputNumber.tsx

```diff
@@ -0,0 +1,105 @@
+import type { ChangeEvent } from 'react'
+import { useCallback, useMemo } from 'react'
+import type { InputProps } from 'baseui/input'
+import { useStyletron } from 'baseui'
+import { Input as BaseUIInput } from 'baseui/input'
+import { Slider as BaseUISlider } from 'baseui/slider'
+
+export interface IInputNumberProps extends Omit<InputProps, 'max' | 'min' | 'step' | 'value' | 'onChange'> {
+  value: number
+  step?: number
+  minimum?: number
+  maximum?: number
+  exclusiveMinimum?: number
+  exclusiveMaximum?: number
+  onChange?: (value: number) => void
+}
+
+export default function InputNumber({
+  value,
+  step = 1,
+  onChange,
+  minimum,
+  maximum,
+  exclusiveMinimum,
+  exclusiveMaximum,
+  ...restProps
+}: IInputNumberProps) {
+  const [css] = useStyletron()
+  const min = useMemo(() => {
+    if (exclusiveMinimum !== undefined)
+      return exclusiveMinimum
+    if (minimum !== undefined)
+      return minimum + step
+    return undefined
+  }, [minimum, exclusiveMinimum])
```

Comment:
It seems that you have reversed these two attributes, exclusive* means not including.

Reply:
fixed